Models/Scraper/Scraper.php

```diff
@@ -134,6 +140,66 @@ private function getCombinedTags($content, array $basicTags)
 		return array_keys($tags);
 	}
 
+	/**
+	 * Get the website description.
+	 * @param string $url Website URL
+	 * @return string Description website
+	 * Resource:
+	 * - https://www.codespeedy.com/get-meta-tags-of-a-web-page-in-php/
+	 * - https://stackoverflow.com/questions/7454644/how-to-get-open-graph-protocol-of-a-webpage-by-php
+	 * Limitations:
+	 * - Shopify sites -> Shopify prevents scraping
+	 */
+
```

Comment:
Just remove this empty line between the beginning of the function and the function documentation (comments) because it is confusing (according to PHPDoc)

Reply:
👍🏼